hack/yq-fixup-yamls.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/sh
+# use yq to reformat yaml files to fit yamllint's syntax validity checks
+#
+# usage: 
+#   yq-fixup-yamls.sh file.yaml 
+#   yq-fixup-yamls.sh dirpath
+#
+YQ=${YQ:-$(command -v yq)}
+${YQ} --version > /dev/null
+
+_yaml_fixup_file() {
+	yaml="$1"
+
+	# use yq as yaml formatter
+	${YQ} eval "${yaml}" > "${yaml}".tmp
+
+	# override original yaml file if differ
+	dif=$(diff -q "${yaml}" "${yaml}".tmp)
+	if [ "${dif}" != "" ]; then
+		mv "${yaml}".tmp "${yaml}"
+	else
+		rm -f "${yaml}".tmp
+	fi
+}
+
+_yaml_fixup_files() {
+	yamls=$(find "$1" -type f -name '*.yml' -or -name '*.yaml')
```

Comment:
Perhaps we should do -iname instead of name for case-insensitive matches? (example: FOO.YAML)

Reply:
Do we have such cases? IMHO, it would be better to rename FOO.YAML --> foo.yaml.

Reply:
The issue is that it won't get caught. I'm not against preferring lowercase, I personally prefer it. So, while I won't insist on it I figured I would bring it up.